core/internal/stream/stream.go

```diff
@@ -116,6 +118,17 @@ func streamLogger(
 			Sentry: sentryClient,
 		},
 	)
+
+	// Log when we receive a shutdown signal
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, syscall.SIGINT)
+	signal.Notify(c, syscall.SIGTERM)
```

Comment:
I think `Notify` can take multiple signals, can't it?

```suggestion
	c := make(chan os.Signal, 1)
	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
```

Reply:
You are correct! Updated